### ISSUE TITLE
panic: could not match regexp with string

### ISSUE THREAD

**Issue by kjk** [elliotchance/c2go#644]
I was trying to transpile [cmark](https://github.com/commonmark/cmark) on Mac OS.

What I did:
* `go get -u github.com/elliotchance/c2go`
* `git clone https://github.com/commonmark/cmark.git`
* `cd cmark`
* `make` : this is needed to generate `buid/src/config.h` etc.
* `c2go transpile -clang-flag -Ibuild/src src/blocks.c`

I get this error:
```
$ c2go transpile -clang-flag -Ibuild/src src/blocks.c
panic: could not match regexp with string
^(?P<address>[0-9a-fx]+) <(?P<position>.*)>(?P<default> Default)?[\s]*$
0x7fee0d905c68 </Users/kjk/src/go/src/github.com/kjk/cmark/src/cmark.h:467:16, col:36> Inherited Default


goroutine 70 [running]:
github.com/elliotchance/c2go/ast.groupsFromRegex(0xc420346410, 0x47, 0xc4206451b3, 0x68, 0x1)
	/Users/kjk/src/go/src/github.com/elliotchance/c2go/ast/ast.go:270 +0x379
github.com/elliotchance/c2go/ast.parseVisibilityAttr(0xc4206451b3, 0x68, 0x12796db)
	/Users/kjk/src/go/src/github.com/elliotchance/c2go/ast/visibility_attr.go:12 +0x62
github.com/elliotchance/c2go/ast.Parse(0xc4206451a4, 0x77, 0x1274be1, 0x5)
	/Users/kjk/src/go/src/github.com/elliotchance/c2go/ast/ast.go:244 +0x2a96
main.convertLinesToNodes(0xc42020b190, 0x272, 0x15fd, 0x0, 0x0, 0x0)
	/Users/kjk/src/go/src/github.com/elliotchance/c2go/main.go:89 +0x198
main.convertLinesToNodesParallel.func1.2(0xc4200882a0, 0xc42024c000, 0xc42020b190, 0x272, 0x15fd, 0x0)
	/Users/kjk/src/go/src/github.com/elliotchance/c2go/main.go:121 +0x59
created by main.convertLinesToNodesParallel.func1
	/Users/kjk/src/go/src/github.com/elliotchance/c2go/main.go:119 +0x19d
```

Not sure which clang is used as I have llvm installed with brew, but probably the one that comes with XCode:
```
$ clang --version
Apple LLVM version 9.0.0 (clang-900.0.39.2)
Target: x86_64-apple-darwin17.4.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```

**Comment by elliotchance** [comment]
Hi @kjk, I've put in some fixes with #645 . Try again with the latest version :)

**Comment by kjk** [comment]
The error is gone, thanks!
